pages/quiz/[slug].tsx

```diff
@@ -80,7 +80,7 @@ function Quiz({ quizTitle, quizQuestions: originalQuizQuestions }: QuizProps) {
         secondsToComplete: timer,
       });
     }
-  }, [quizQuestions.length, quizRecord.length])
+  }, [quizRecord, quizSubmitted, quizTitle, timer, quizQuestions.length])
```

Comment:
Same for this - was it needed for updating Next?

Reply:
Similarly, felt it was small enough.